### ISSUE TITLE
Move Decorators written for tutorial to ART

### ISSUE THREAD

**Issue by SebChw** [SebChw/Actually-Robust-Training#178]
in the first tutorial we introduce few decorators. Why don't add them to the ART directly and let user use them straightaway. In V0.4 we can write even more of them.

1.  LogginDecorator - this will just log some basic statistics of tensors inputted to the network
2. EvaluationSaver - this will show logits changes with time
3. Batch Saver - this will just save images before they are inputted to the network

It's just about moving them from tutorial to art not implementing them
